tedium.js

```diff
@@ -27,7 +29,41 @@ const path = require('path');
 const ProgressBar = require('progress');
 const rimraf = require('rimraf');
 const cleanup = require('./cleanup-passes');
-
+const hydrolysis = require('hydrolysis');
+const pad = require('pad');
+const cliArgs = require("command-line-args");
+const cli = cliArgs([
+  {
+    name: "help",
+    type: Boolean,
+    alias: "h",
+    description: "Print usage."
+  },
+  {
+    name: "max_changes",
+    type: (x) => {
+      if (!x) {
+        return 0;
+      }
+      if (/^[0-9]+$/.test(x)) {
+        return parseInt(x, 10);
+      }
+      throw new Error(`invalid max changes, expected an integer: ${x}`);
+    },
+    alias: "c",
+    description: "The maximum number of repos to push. Defualt: 0",
+    defaultValue: 0
+  },
+]);
+const opts = cli.parse();
+
+if (opts.help) {
+  console.log(cli.getUsage({
+    header: "tedium is a friendly bot for doing mass changes to Polymer repos!",
```

Comment:
![Image of happy flower swaying](http://orig11.deviantart.net/3e8a/f/2015/293/c/5/flowey_large_by_sovanjedi-d9du6ic.gif)